pkg/oc/admin/migrate/scc/scc.go

```diff
@@ -0,0 +1,240 @@
+package scc
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	"strings"
+
+	extensions "k8s.io/api/extensions/v1beta1"
+	rbacv1 "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	kapi "k8s.io/kubernetes/pkg/apis/core"
+	rbacv1helper "k8s.io/kubernetes/pkg/apis/rbac/v1"
+	"k8s.io/kubernetes/pkg/kubectl/cmd/templates"
+	kcmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
+
+	cmdutil "github.com/openshift/origin/pkg/cmd/util"
+	"github.com/openshift/origin/pkg/oc/cli/util/clientcmd"
+	securityapi "github.com/openshift/origin/pkg/security/apis/security"
+	securitytypedclient "github.com/openshift/origin/pkg/security/generated/internalclientset/typed/security/internalversion"
+
+	"github.com/spf13/cobra"
+)
+
+// pspPriorityAnnotationKey is a name of the annotation that could be present on PodSecurityPolicy and specifies its priority.
+// It should be a number. An empty value means 0 priority. Higher values means that the policy has higher priority.
+// This is an equivalent of the SCC.Priority field but applies to PSP and it's required for seamless transition.
+const pspPriorityAnnotationKey = "security.openshift.io/psp-priority"
+
+var (
+	internalMigrateSCCShort = "Converts SCCs to similar PSPs"
+	internalMigrateSCCLong  = templates.LongDesc(`
+		Converts SecurityContextConstraints to the equivalent PodSecurityPolicy objects.
+		For each SCC it also creates a ClusterRole with the same name and ClusterRoleBinding
+		for every user or group that SCC has.
+
+		NOTE: it does not modify anything by default.`)
+
+	internalMigrateSCCExample = templates.Examples(`
+		# Output to the console PodSecurityPolicy, ClusterRoles, and ClusterRoleBinding objects
```

Comment:
`Perform a dry-run of migrating SCC to ...`

Additionally add:
```
# To actually perform the migration, the confirm flag must be appended
%[1]s --confirm
```

Reply:
If there are any other flag combinations, provide examples as well.

Reply:
Is it a request to add `--confirm` option and create these objects?

@simo5 Do we need such behavior?

Reply:
I think we could add a --confirm option ... once we know what the final migration strategy is.
ATM we do not know if we want to keep SCCs side by side with PSP or replace.

@soltysh this tool is exploratory at this stage, and used to test out what strategies we can use.

Reply:
I don't see any reasons not to have the options to replace/add and whatever approach you're testing. In general I'd like this command not to be any different than other commands we have in the migration path.